src/layouts/partials/footer.html

```diff
@@ -5,7 +5,9 @@
     <div class="footercolumns width1100">
         <div class="footerrows w-row">
             <div class="footerrowscolumn1 w-col w-col-3 w-col-medium-3">
-                <a href="https://github.com/decred" target="_blank" rel="noopener noreferrer" class="footerblocklinkdev">{{ T "general_copyright" }}</a>
+                <a href="https://github.com/decred" target="_blank" rel="noopener noreferrer" class="footerblocklinkdev footer-margin-bottom">{{ T "general_copyright" }}</a>
+                <a href="https://time.decred.org" target="_blank" class="footer-inline footerblocklink transition footer-margin-top footer-margin-left">{{ T "nav_time" }}</a>
+                <a href="https://bounty.decred.org" target="_blank" class="footer-inline footerblocklink transition">{{ T "nav_bounty" }}</a>
             </div>
```

Comment:
what is the reasoning for adding time and bounty?

Reply:
It does feel a little iffy to me maybe more development projects around 6 can be listed with it in a smaller font. Bounty makes sense there I think, as it let's people know it is available.

@zhizhongzhiwai was the one that suggested it on the task page.

Reply:
Use https://timestamp.decred.org instead of time.

Reply:
Updated